src/lib/list/list.ts

```diff
@@ -70,6 +93,12 @@ export class MdListCssMatStyler {}
 })
 export class MdNavListCssMatStyler {}
 
+@Directive({
+  selector: 'md-selection-list, mat-selection-list',
+  host: {'class': 'mat-selection-list'}
+})
+export class MdSelectionListCssMatStyler {}
```

Comment:
You shouldn't need this class; you can add the `mat-selection-list` class in `MdSelectionList`